tuhi.py

```diff
@@ -253,18 +253,28 @@ def _on_bluez_discovery_stopped(self, manager):
         # restart discovery if some users are already in the listening mode
         self._on_listening_updated(None, None)
 
-    def _on_bluez_device_updated(self, manager, bluez_device):
-        if bluez_device.vendor_id != WACOM_COMPANY_ID:
+    def _on_bluez_device_updated(self, manager, bluez_device, event=True):
```

Comment:
API-wise, I'd really prefer the `event` parameter to be a `type` parameter and an enum. Much more readable having things like `if type == PRESENT_ON_START` or something than the quite ambiguous term "event"

Reply:
That's what reviews are for. I'll try to work on it today.